package.json

```diff
@@ -116,6 +133,20 @@
     "styled-components": "^3.4.9",
     "yup": "^0.26.6"
   },
+  "peerDependencies": {
+    "@babel/runtime": ">= 7.1.2",
+    "classnames": ">= 2.2.6",
+    "flexboxgrid": ">= 6.3.1",
+    "formik": ">= 1.3.1",
+    "next": ">= 7.0.1",
```

Comment:
What is the need for next in this setup ?

Reply:
The Anchor Component uses Next/Link but we can take it out.

Reply:
We should take it out. React link should be fine as we are already including react as a peerDependency